var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -29,6 +29,8 @@ class Cmake(Package):
     license("BSD-3-Clause")
 
     version("master", branch="master")
+    version("3.29.2", sha256="36db4b6926aab741ba6e4b2ea2d99c9193222132308b4dc824d4123cb730352e")
+    version("3.29.0", sha256="a0669630aae7baa4a8228048bf30b622f9e9fd8ee8cedb941754e9e38686c778")
```

Comment:
```suggestion
```

Reply:
If we want to preserve 3.29.0, I think we should add it in the list of deprecated versions.